### PR TITLE
feat(SelectCertificateModal): add component to handle SSL certificate select in Electron apps

### PR DESCRIPTION
**Pull request by MatveyNeyman**

### CODE REVIEW

File: src/components/CreateNewModal/CreateNewModal.js [hunk 1]
```diff
@@ -283,7 +283,6 @@ class CreateNewModal extends PureComponent<Props> {
         </ModalBody>
         <ModalFooter className={styles.footer}>
           <Button
-            className={styles.halfButton}
```
Comment (alexboyko) [inline src/components/CreateNewModal/CreateNewModal.js]: Is it accidental removal?

Comment (MatveyNeyman) [reply src/components/CreateNewModal/CreateNewModal.js]: No, found out that there is no halfButton property anywhere else in the code

File: src/components/Radio/Radio.js [hunk 2]
```diff
@@ -48,7 +48,11 @@ class Radio extends PureComponent<Props> {
 
   renderChildren() {
     const { children } = this.props;
-    if (!children) {
+
+    if (!children || typeof children !== 'object') {
+      // eslint-disable-next-line
+      console.error('Error: Radio component must have a child of React component type');
```
Comment (nkt) [inline src/components/Radio/Radio.js]: This must be validated by flow

Comment (MatveyNeyman) [reply src/components/Radio/Radio.js]: Fixed

**Pull request merged**